services/common/src/redux/utils/Validate.spec.ts

```diff
@@ -0,0 +1,496 @@
+import {
```

Comment:
I deleted the Validate.js files in core & ms (and updated references), so I brought over the test file to common.